backend/src/interface/image_review/mod.rs

```diff
@@ -1 +1,10 @@
 //! This interface allows starting the process of checking the existence of images linked to meals.
+
+use async_trait::async_trait;
+
+/// This interface allows starting the image review process to check for deleted images at the image hoster.
+#[async_trait]
+pub trait ImageReviewScheduling {
```

Comment:
Here, clippy wants us to remove package name redundancy in the trait name. Should we follow or ignore that?

Reply:
Could not suppress the error